modes/longitudinal/src/toolbarButtons.js

```diff
@@ -307,4 +307,19 @@ export default [
       type: 'secondary',
     },
   },
+  {
+    id: 'Clear',
+    type: 'ohif.action',
+    config: {
+      groupName: 'primaryTool',
+    },
+    props: {
+      isActive: false,
+      icon: 'old-trash',
```

Comment:
hahahaha

Reply:
I didn't know we had an icon of me in OHIF.